libraries/ms-common/src/stm32f0xx/spi.c

```diff
@@ -11,14 +11,10 @@ typedef struct {
 
 static SPIPortData s_port[SPI_MCU_NUM_PORTS] = {
   {
-    .rcc_cmd = RCC_APB2PeriphClockCmd,
-    .periph = RCC_APB2Periph_SPI1,
-    .base = SPI1
+      .rcc_cmd = RCC_APB2PeriphClockCmd, .periph = RCC_APB2Periph_SPI1, .base = SPI1,
```

Comment:
Was this supposed to be one line? It's kinda hard to figure out what the rule for initializers is. Doesn't seem to be consistent even though you're ending it with a comma?